test/beforeHeadersSend.test.js

```diff
@@ -97,14 +115,16 @@ describe("redemption attempted", () => {
     test("redemption header added", () => {
         setSpendFlag(url.host, true);
         setSpentUrl(url.href, false);
+        workflow.__set__("REDEEM_METHOD", "reload");
         let redeemHdrs = beforeSendHeaders(details, url);
         let reqHeaders = redeemHdrs.requestHeaders;
-        expect(reqHeaders).toBeTruthy();
-        expect(reqHeaders[0].name == "challenge-bypass-token").toBeTruthy();
-        expect(reqHeaders[0].value == b64EncodedToken).toBeTruthy();
+        expect(getSpendFlag(url.host)).toBeFalsy();
         expect(getSpendId([details.requestId])).toBeTruthy();
         expect(getSpentUrl([url.href])).toBeTruthy();
         expect(getSpentTab([details.tabId]) == url.href).toBeTruthy();
+        expect(reqHeaders).toBeTruthy();
+        expect(reqHeaders[0].name == "challenge-bypass-token").toBeTruthy();
```

Comment:
Does this string `"challenge-bypass-token"` must be a constant ? or it is safe to use the literal value?